posthog/api/organization_invite.py

```diff
@@ -62,13 +79,32 @@ def validate_invites(self, data: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
 
     def create(self, validated_data: Dict[str, Any]) -> Dict[str, Any]:
         output = []
+        name_count: int = 0
+        current_invite_count = OrganizationInvite.objects.filter(
+            organization_id=self.context["organization_id"],
+        ).count()
 
         with transaction.atomic():
             for invite in validated_data["invites"]:
+                self.context["bulk_create"] = True
                 serializer = OrganizationInviteSerializer(data=invite, context=self.context)
                 serializer.is_valid(raise_exception=False)  # Don't raise, already validated before
                 output.append(serializer.save())
 
+                if invite["first_name"]:  # For analytics
+                    name_count = name_count + 1
+
+        report_bulk_invited(
+            self.context["request"].user.distinct_id,
+            invitee_count=len(validated_data["invites"]),
+            name_count=name_count,
```

Comment:
This is atm distributed across so many lines. Let's keep analytics code as centralized as possible - it makes everything too messy otherwise.

```
sum(1 for invite in validated_data["invites"] if invite["first_name"])
```